src/main/scala/nl.knaw.dans.easy.bagstore/package.scala

```diff
@@ -86,4 +86,13 @@ package object bagstore {
       }
     }
   }
+
+  // TODO: canditates for dans-scala-lib?
```

Comment:
Maybe we should, but then in the style of `path.listDirs`, I propose. Perhaps we can look into this next week. I got some more I/O stuff that I'm thinking of putting in the lib.

Besides that, I've got some branches with other proposals for the lib hanging around locally, som maybe we could add those in the same release?